### PR TITLE
MDOT Confirm Page and Title Change

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description
Creating initial set up of confirm page and swapping title to "Order hearing aid batteries and accessories".

## Testing done
Using hard coded data until our API is set up.

## Acceptance criteria
- [ ] Given that I have completed the Form 2346 application
- [ ] When I arrive at the confirmation page
- [ ] Then I should see a "your order has been submitted" section,  a "your order details" section, a "how long will it take to receive order" section, and a "your order has been submitted" section

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -15,36 +19,95 @@ export class ConfirmationPage extends React.Component {
...
+          content={
+            <section>
+              <p>
+                <strong>Order Date</strong>:{' '}
+                {new Intl.DateTimeFormat('en-US', {
+                  month: 'long',
+                  day: 'numeric',
+                  year: 'numeric',
+                }).format(date)}
+              </p>
```
Comment (Mottie) [inline src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx]: I'm not sure if we should be using `Intl.DateTimeFormat` yet. It is supported in most browsers, including IE11, but I don't think it's standard practice. Usually `moment` is used for this type of formatting.

**Review by Mottie — APPROVED** [review]

**Review by mojaray2k — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]
If you merge master in your PR should pass the security check.

**Comment by mr0sari0** [comment]
> If you merge master in your PR should pass the security check.

Hmm it tells me I am up to date with master already.

**Comment by bkjohnson** [comment]
I'm guessing your local `master` wasn't up to date with the remote. I went ahead and did it for you.

**Comment by mr0sari0** [comment]
> I'm guessing your local `master` wasn't up to date with the remote. I went ahead and did it for you.

Thanks!

**Pull request merged**